cmd/notation/key.go

```diff
@@ -91,7 +87,7 @@ func keyUpdateCommand(opts *keyUpdateOpts) *cobra.Command {
 		opts = &keyUpdateOpts{}
 	}
 	command := &cobra.Command{
-		Use:     "update [name]",
+		Use:     "update [flags] <key_name>",
```

Comment:
update --default <key_name>
Looks like `update` command is only for updating `default` key name?

Reply:
According to the [spec](https://github.com/notaryproject/notation/blob/main/specs/commandline/key.md#update-the-default-signing-key), yes.